### DISCUSSION TITLE
Multi-line input

### DISCUSSION THREAD

**Discussion by ghost** [vishalxl/nostr_console#85] [Ideas]
Sending long messages is difficult

**Comment by vishalxl** [comment]
hi, support long messages are a factor of which version of terminal/command line you are using. 

As of today, I can sent long messages using the windows terminal ( on windows 11) , and ubuntu 22. 

An example post with long message is here: https://primal.net/e/note1dwf60zkvp7tzl965j04k4ynzpsgyq9su3zyzfcgz4erruxx456hqpjxqwg 
made on Ubuntu 22. 

Overall, long messages are supported ( longer than 256 bytes). About 1+ years ago, such messages were not possible on Windows Terminal at least.
